apps/sms/views/shared/js/contacts.js

```diff
@@ -246,13 +246,13 @@
 
     findExact: function contacts_findExact(filterValue) {
       return this.findBy({
-        filterBy: ['givenName', 'familyName'],
+        filterBy: ['tel', 'givenName', 'familyName'],
         filterOp: 'contains',
```

Comment:
so, I checked closed, and for the phone number, we need the "match" operation.

Actually this is already implemented by `Contacts.findByPhoneNumber` and I don't think we can do better than doing 2 calls. And actually it's likely better to call `Contacts.findByAddress` to handle the email case as well.

So here is what I think could work. In https://github.com/mozilla-b2g/gaia/blob/09e233d1476e8745f192f13259d7b16662bb2e7e/apps/sms/views/conversation/js/conversation.js#L2618, replace this line by:
```javascript
  return Promise.all([
    Contacts.findExact(fValue),
    Contacts.findByAddress(fValue)
  ]).then(([ result1, result2 ]) => result1.concat(result2));
```
(thanks to your bug 1153888 we don't need to check that result1 or result2 are not null :) ).

Does it make sense ?